src/style.css

```diff
@@ -0,0 +1,116 @@
+.page {
+  font-family: "roboto", sans-serif;
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: auto;
+}
+
+.card__photo {
+  background-image: url(./images/imac.jpeg);
+  background-size: cover;
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px;
```

Comment:
remove margin from card, center it with flex properties (card should be a flex container)